rewrite-java/src/main/java/org/openrewrite/java/AddOrUpdateAnnotationAttribute.java

```diff
@@ -70,6 +72,14 @@ public String getDescription() {
     @Nullable
     Boolean addOnly;
 
+    @Option(displayName = "Append array",
+            description = "If the attribute is an array, setting this option to `true` will append the value(s). " +
+                          "In conjunction with `addOnly`, it is possible to control duplicates:" +
```

Comment:
Note that String concatenation means these lines would be directly appended. Best to have a space after the colon then.
```suggestion
                          "In conjunction with `addOnly`, it is possible to control duplicates: " +
```

Reply:
Fixed